bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/thing/shellyGen1_sensor.xml

```diff
@@ -487,6 +461,65 @@
 		</state>
 	</channel-type>
 
+	<channel-type id="sensorExtTemp">
+		<item-type>Number:Temperature</item-type>
+		<label>@text/channel-type.shelly.sensorExtTemp.label</label>
+		<description>@text/channel-type.shelly.sensorExtTemp.label</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
+		<state readOnly="true" pattern="%.2f %unit%">
```

Comment:
Does it make sense to display the temperature with two digits after the decimal point?
```suggestion
		<state readOnly="true" pattern="%.1f %unit%">
```

Reply:
fixed

Reply:
It's still 2?